specification/maps/data-plane/Weather/stable/2.0/weather.json

```diff
@@ -0,0 +1,3869 @@
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/JsonFormat"
+          },
+          {
+            "$ref": "#/parameters/WeatherYear"
+          },
+          {
+            "$ref": "#/parameters/RequiredWeatherBasinId"
+          },
+          {
+            "$ref": "#/parameters/RequiredWeatherGovernmentId"
+          },
+          {
+            "$ref": "#/parameters/WeatherDetails"
+          },
+          {
+            "$ref": "#/parameters/WeatherRadiiGeometry"
+          },
+          {
+            "$ref": "#/parameters/WeatherDataUnit"
+          },
+          {
+            "name": "current",
+            "description": "When true, return the current storm location",
+            "x-ms-client-name": "includeCurrentStorm",
+            "in": "query",
+            "type": "boolean",
+            "default": false
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/StormLocationsResult"
+            }
+          },
+          "default": {
+            "$ref": "../../../Common/preview/1.0/common.json#/responses/default"
+          }
+        }
+      }
+    },
+    "/weather/airQuality/current/{format}": {
+      "get": {
+        "description": "**Get Current Air Quality**\n\n**Applies to**: Gen 1 (S0 and S1) and Gen 2 pricing tiers.\n\nGet detailed information about the concentration of pollutants and overall status for current air quality. Information includes, pollution levels, air quality index values, the dominant pollutant, and a brief statement summarizing risk level and suggested precautions.",
+        "operationId": "Weather_GetCurrentAirQuality",
+        "x-ms-examples": {
+          "Get Current Air Quality": {
+            "$ref": "./examples/GetCurrentAirQuality.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/ClientId"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/JsonFormat"
+          },
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/Language"
+          },
+          {
+            "$ref": "#/parameters/Position"
+          },
+          {
+            "$ref": "#/parameters/Pollutants"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/AirQualityResult"
+            }
+          },
+          "default": {
+            "$ref": "../../../Common/preview/1.0/common.json#/responses/default"
+          }
+        }
+      }
+    },
+    "/weather/airQuality/forecast/daily/{format}": {
```

Comment:
Similar question here - could hourly, daily, etc.. be extensible enum parameters?

Reply:
The response formats of daily and hourly forecase are slightly different.
So I think we might need to separate them.